package.json

```diff
@@ -88,6 +88,7 @@
     "image-webpack-loader": "3.6.0",
     "imports-loader": "0.7.1",
     "jest": "22.1.4",
+    "jest-canvas-mock": "1.0.2",
```

Comment:
So that the doughtnut chart doesn't throw errors about `getContext` not being defined.